thesslstore_plugin.php

```diff
@@ -126,6 +130,33 @@ private function upgrade1_2_2()
         }
     }
 
+    /**
+     * Upgrades to v1.3.0 of the plugin
+     * Removes service synchronization cron task that was moved to module
+     */
+    private function upgrade1_3_0()
+    {
+        Loader::loadModels($this, ['CronTasks']);
+
+        $cron_tasks = $this->getCronTasks();
+
+        // Remove the cron task
+        foreach ($cron_tasks as $task) {
```

Comment:
You shouldn't be iterating over any cron tasks, especially now that you've removed the one you intend to delete from ::getCronTasks. Just delete the cron task you know shouldn't exist.

I think a call to CronTasks::deleteTask should suffice for deleting the cron task and the cron task runs for all companies.